core/domain/stats_jobs_one_off.py

```diff
@@ -65,9 +65,6 @@ def map(playthrough_model):
                     trying to create the model as a domain object, or None if no
                     error occurred.
         """
-        if playthrough_model.deleted:
```

Comment:
playthrough_model is not a subclass of versioned_model, i.e there is no option for only marking the model as deleted(the model instance will be completely deleted from the datastore) and hence this line cannot be reached